config/_common_search_paths.py

```diff
@@ -1,37 +1,36 @@
 import os
 
 def _check_dirs(search_paths):
-        for search_path in search_paths:
-                if os.path.isdir(search_path):
-                        return search_path
-        return None
+    for search_path in search_paths:
+        if os.path.isdir(search_path):
+            return path
```

Comment:
```suggestion
            return search_path
```
I think this undoes a change @mabruzzo made in PR#65.

Reply:
Oh yes, thanks. That was unintentional.